flexmeasures/cli/data_show.py

```diff
@@ -449,6 +449,15 @@ def list_data_sources(source: DataSource | None = None, show_attributes: bool =
     required=False,
     help="Resolution of the data in ISO 8601 format. If not set, defaults to the minimum resolution of the sensor data. Note: Nominal durations like 'P1D' are converted to absolute timedeltas.",
 )
+@click.option(
+    "--combine-legend",
```

Comment:
Is there a test (skipped in GH) for this CLI command already? If so, let's extend it to cover the new option. If not, let's find out whether it would be straightforward to create one.